src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
```

Comment:
- [ ] Good job so far 👍 You are required to use class-based components, as a result it would be great if you could convert the `App` to a class-based component 💯